src/ophys_etl/workflows/pipeline_module.py

```diff
@@ -108,8 +116,14 @@ def dockerhub_repository_name(self) -> str:
 
     @property
     @abc.abstractmethod
-    def inputs(self) -> Dict:
-        """Input args to module"""
+    def module_schema(self) -> DefaultSchema:
+        """Argschema to validate module_args"""
+        raise NotImplementedError
+
+    @property
+    @abc.abstractmethod
+    def module_args(self) -> Dict:
```

Comment:
They can, but it would require the dev to implement the validation line. 
This might pose issues if it was forgotten. I wanted to separate the validation line from the property implementation.

Reply:
I'm not following. Just pick a name of `inputs` or `module_args`, and don't use both as it's confusing to have 2 attributes with different names that have essentially the same value